code/modules/transport/pods/MainWeapon.dm

```diff
@@ -1173,6 +1180,30 @@ TYPEINFO(/obj/item/shipcomponent/mainweapon/constructor)
 	dissipation_delay = 14
 	projectile_speed = 42
 
+/datum/projectile/laser/light/pod/burst
+	damage = 25
+	//shot_number = 3 - this would be better but is buggy
```

Comment:
Hmm, explain?

Reply:
Just from messing around, shots would sometimes fire in a single direction, sometimes whichever direction the pod is facing when as it turns when firing, and sometimes in seeminly random directions like behind the pod

Reply:
That seems worth fixing, rather than re-implementing the entire thing here.